src/test/java/fi/metatavu/metaform/server/test/functional/tests/ExportXlsxTestsIT.kt

```diff
@@ -23,53 +32,24 @@ import java.io.IOException
-//            assertNotNull(tableSheet)
-//            assertEquals("Text table field", tableSheet.getRow(0).getCell(0).getStringCellValue())
-//            assertEquals("Text 1", tableSheet.getRow(1).getCell(0).getStringCellValue())
-//            assertEquals("Text 2", tableSheet.getRow(2).getCell(0).getStringCellValue())
-//            assertEquals(10.0, tableSheet.getRow(1).getCell(1).getNumericCellValue(), 0)
-//            assertEquals(20.0, tableSheet.getRow(2).getCell(1).getNumericCellValue(), 0)
-//            assertEquals(30.0, tableSheet.getRow(3).getCell(1).getNumericCellValue(), 0)
-//            assertEquals("SUM(B2:B3)", tableSheet.getRow(3).getCell(1).getCellFormula())
-//        }
-//    }
-//
-//    @Test
-//    @Throws(Exception::class)
-//    fun testExportXlsxTableScripted() {
-//        TestBuilder().use { builder ->
-//            val parsedMetaform: Metaform = builder.metaformAdmin.metaforms().readMetaform("scripted-xlsx-table")
-//            val metaform: Metaform = builder.metaformAdmin.metaforms().create(parsedMetaform)
-//            val tableData: List<Map<String, Any>> = Arrays.asList(createSimpleTableRow("Text 1", 10.0), createSimpleTableRow("Text 2", 20.0))
-//            val replyData: Map<String, Any> = HashMap()
-//            replyData.put("table", tableData)
-//            val replyWithData: Reply = builder.test1.replies().createReplyWithData(replyData)
-//            builder.test1.replies().create(metaform.id, null, ReplyMode.REVISION.toString(), replyWithData)
-//            val workbook = getXlsxReport(metaform, builder.metaformAdmin.token())
-//            val simpleSheet: Sheet = workbook.getSheet("Simple")
-//            assertNotNull(simpleSheet)
-//            assertEquals("Sum field", simpleSheet.getRow(0).getCell(1).getStringCellValue())
-//            assertEquals("SUM('Table field - 1'!B1:'Table field - 1'!B3)", simpleSheet.getRow(1).getCell(1).getCellFormula())
-//        }
-//    }
+    @Test
+    @Throws(Exception::class)
+    fun testExportXlsxTable() {
+        TestBuilder().use { builder ->
+            val parsedMetaform = builder.systemAdmin.metaforms.readMetaform("simple-table")
+            val metaform = builder.systemAdmin.metaforms.create(parsedMetaform!!)
+            val tableData: List<Map<String, Any>> = listOf(createSimpleTableRow("Text 1", 10.0), createSimpleTableRow("Text 2", 20.0))
+            val replyData: HashMap<String, Any> = HashMap()
+            replyData["table"] = tableData
+            val replyWithData: Reply = builder.test1.replies.createReplyWithData(replyData)
+            builder.test1.replies.create(metaform.id!!, null, ReplyMode.REVISION.toString(), replyWithData)
+            val workbook = getXlsxReport(metaform, builder.systemAdmin.token)
+            val simpleSheet: Sheet = workbook.getSheet("Simple")
+            assertNotNull(simpleSheet)
+            val tableSheet: Sheet = workbook.getSheet("Table Field - 1")
+            assertNotNull(tableSheet)
```

Comment:
Add more thorough tests that actually do assertions about the sheet content.

Reply:
Fixed